IPython/html/static/edit/less/edit.less

```diff
@@ -1,3 +1,10 @@
+.header_edit {
+}
+
+.site_edit {
+    height: 88.5%;
```

Comment:
Why is it 88.5%? Where did you get that number? Why not 100%? If it's accounting for the header, any time you account for a fixed height with a percentage, the math is wrong when the window changes height. `calc` lets you do this correctly, e.g. `calc(100vh - 64px)` is the full height of the viewport minus 64px.

For an example, make the window fairly short (on my monitor, less than 800px), and this PR will calculate the wrong height and add extra scrollbars not present on master:

![screen shot 2015-03-31 at 14 37 39](https://cloud.githubusercontent.com/assets/151929/6929909/886e1fc8-d7b3-11e4-8fcf-4f926eef4f8c.PNG)